modules/account-lib/test/unit/mpc/tss.ts

```diff
@@ -139,27 +136,26 @@ describe('TSS EDDSA key generation and signing', function () {
       const derive1 = MPC.deriveUnhardened(commonKeychain, path);
       const subkey = MPC.keyDerive(A.uShare, [B.yShares[1], C.yShares[1]], path);
       const derive2 = MPC.deriveUnhardened(commonKeychain, path);
+      const derivedPk = Buffer.from(derive1, 'hex').slice(0, 32).toString('hex');
```

Comment:
Dumb question, but why do we need to convert to buffer and then back to hex? 
Instead of something like derivedPk = derive2.slice(0, 64) since its already in hex

Reply:
mistake on my side - i thought i observed some flakiness in my test when using `string.slice(0, 64)`
updated one usage to use `string.slice(0, 64)` and left others as Buffer. (i guess to demonstrate that they're equivalent).

Reply:
Alright, thanks for checking. I guess on the platform side i'll just use `string.slice(0, 64)` then